src/shared/svelte-compile-warns/index.ts

```diff
@@ -401,17 +401,37 @@ function* transformScripts(context: RuleContext, text: string) {
 	}
 }
 
+function hasTagOption(program: AST.SvelteProgram) {
+	return program.body.some((body) => {
+		if (body.type !== 'SvelteElement' || body.kind !== 'special') {
+			return false;
+		}
+		if (body.name.name !== 'svelte:options') {
+			return false;
+		}
+
+		return Boolean(findAttribute(body, 'tag'));
+	});
+}
+
 /**
  * Get compile warnings
  */
-function getWarningsFromCode(code: string): {
+function getWarningsFromCode(
+	code: string,
+	context: RuleContext
+): {
 	warnings: Warning[];
 	kind: 'warn' | 'error';
 } {
 	try {
 		const result = compiler.compile(code, {
 			generate: false,
-			...(semver.satisfies(compiler.VERSION, '>=4.0.0-0') ? { customElement: true } : {})
+			...(semver.satisfies(compiler.VERSION, '>=4.0.0-0')
+				? { customElement: true }
```

Comment:
[ask]

Why always we can set `{ customElement: true }` if Svelte version is 4?

Reply:
In v3, if `customElement:true` is set, a warning will be returned if `<svelte:options tag>` is missing, but in v4, there will be no warning even if `<svelte:options customElement>` is missing.